app/assets/stylesheets/views/_dit-landing-page.scss

```diff
@@ -43,13 +37,26 @@ $dark-grey: #282828;
   }
 }
 
+.dit-landing-page_header-text-container {
+  background-color: $dark-red;
+  box-shadow: 15px 0 0 0 $dark-red;
```

Comment:
Where does the shadow appear?

Reply:
We're using the `box-shadow` as a technique to render a border outside the element boundaries and we do this to cover the standard margin we have after the `govuk-grid-column-two-thirds` container.

<img width="1440" alt="Screenshot 2020-12-08 at 17 07 21" src="https://user-images.githubusercontent.com/788096/101517064-30a65d00-3978-11eb-95b4-89af8f985d21.png">

Reply:
Yeah, I noticed it "extends the redness" a bit to the right. Would we expect to see some "white" if we removed it? It's a minor thing - I'm just struggling to understand why we need it, given that it also looks OK to me without.

Thanks for the diagram ⭐.

Reply:
If we remove it we'll see more of the background image (the one that appears on the right-hand side). We need it to align the text container with the search box at the top (and with any content sitting in the one-third column; for example the 'keep business moving' logo or the image down below the page).

Reply:
Ahhh, so _that's_ what it's for. Thanks for explaining. Could we add that to the commit? I think it would be helpful for someone in the future, in case it's not obvious to someone else too.